introduce androidx support on the android docs (#1805)
getsentry/sentry-docs: src/collections/_documentation/platforms/android/index.md

If you experience issues like `Could not find class` on devices running the `Dalvik VM`, you may expand the above rules to keep the necessary classes in the main dex file.

### AndroidX Support

> We're using the AndroidX libraries for detecting when the Application is either on background or foreground, this is necessary for having an accurate result across all the Android OS versions.

Reviewer: ```suggestion
We're using the AndroidX libraries for detecting when the Application is either in background or in foreground, this is necessary for having an accurate result across all the Android OS versions.
```

> We're using the AndroidX libraries for detecting when the Application is either in background or in foreground, this is necessary for having an accurate result across all the Android OS versions.

Reviewer 2: ```suggestion
We're using the AndroidX libraries for detecting when the Application is either in the background or in the foreground, this is necessary for having an accurate result across all the Android OS versions.
```

> We're using the AndroidX libraries for detecting when the Application is either in the background or in the foreground, this is necessary for having an accurate result across all the Android OS versions.

Reviewer 3: ```suggestion
Sentry uses the AndroidX libraries for detecting when the Application is either in the background or in the foreground. This is necessary for having an accurate result across all the Android OS versions.
```

> We check on runtime for its availability, so if you're not using the AndroidX libraries, you can remove them from Sentry's transitive dependencies.

Reviewer: ```suggestion
We check during runtime for availability, so if you're not using the AndroidX libraries, you can remove them from Sentry's transitive dependencies.
```

> We check during runtime for availability, so if you're not using the AndroidX libraries, you can remove them from Sentry's transitive dependencies.

Reviewer 2: ```suggestion
We check at runtime for availability, so if you're not using AndroidX libraries, you can remove them from Sentry's transitive dependencies.
```
